ajax/libs/gif.js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gif.js",
+  "description": "JavaScript GIF encoding library",
+  "filename": "gif.min.js",
+  "author": "Johan Nordberg <code@johan-nordberg.com>",
+  "homepage": "http://jnordberg.github.io/gif.js/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jnordberg/gif.js.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "gif",
+    "animation",
+    "encoder"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jnordberg/gif.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "gif*"
```

Comment:
Let's use the pattern `**/*`
As usual, we use `**/*` in `dist` as the basePath.
Thank you.